devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -489,6 +507,35 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         }
     }
 
+
+    yarp::os::Bottle& fixedRotationGroup = config.findGroup("FIXED_SENSOR_ROTATION");
+    if (!fixedRotationGroup.isNull()) {
+        for (size_t i = 1; i < fixedRotationGroup.size(); ++i) {
+            if (!(fixedRotationGroup.get(i).isList() && fixedRotationGroup.get(i).asList()->size() == 2)) {
+                yError() << LogPrefix
+                        << "Childs of FIXED_SENSOR_ROTATION must be lists of 2 elements";
+                return false;
+            }
+            yarp::os::Bottle* list = fixedRotationGroup.get(i).asList();
+            std::string key = list->get(0).asString();
+            yarp::os::Bottle* listContent = list->get(1).asList();
+
+            if (!(    (listContent->size() == 9)
```

Comment:
This is a bit vague to follow without any additional comments about the list of values passed in the configuration

Reply:
I added a comment, at this stage we are passing 9 values since we pass the full transform matrix in the configuration file.